.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Report a suspected bug or incompatibility in RCall.jl
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please try to create as small a reproduction as possible.
+
+ * How did you install RCall.jl and configure the R location?
+ * Did you install any R packages or other packages? Using what method? 
+ * Finally what code did you run?
+
+Please include this information even if it relates to an earlier issue which has been closed.
+
+**Expected and actual behavior**
+Describe what happened versus what you expected to happen.
+
+**Version information**
+Please include the output of `using RCall; RCall.versioninfo()` if possible. Otherwise please include `Main.versioninfo()` from Julia and `sessionInfo()` from R.
```

Comment:
```suggestion
Please include the output of `using RCall; RCall.versioninfo()` if possible. Otherwise please include `versioninfo()` from Julia and `sessionInfo()` from R.
```

No need to qualify stuff in `Main` when running interactively and if you're not running interactively, then you'll need to load InteractiveUtils ...

Reply:
Actually, on second thought, I think we should just simplify this a bit:

````suggestion
Please include the output of running the follow commands in the REPL:

```julia
using RCall
reval("sessionInfo()")
versioninfo()
```
````

And then we can cut out the additional `versioninfo` file below (more on that down there)